core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreConstants.java

```diff
@@ -106,10 +106,13 @@
         //set Advanced properties
         setAdvancedProperty("IsBulkImportSupported", "Is Bulk Import Supported", "false",
                 "Support Bulk User Import Operation for this user store",
-                new Property[] { USER.getProperty(), BOOLEAN.getProperty(), FALSE.getProperty() });
+                new Property[] { USER.getProperty(), BOOLEAN.getProperty(), TRUE.getProperty() });
         setAdvancedProperty(JDBCRealmConstants.DIGEST_FUNCTION, "Password Hashing Algorithm", "SHA-256",
                 UserStoreConfigConstants.passwordHashMethodDescription,
                 new Property[] { USER.getProperty(), STRING.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty("Hash.algorithm.props","UserStore Hashing Configurations", "null",
```

Comment:
We should not pass `"null"` here since it will be displayed in the UI

Reply:
resolved